source/dagger/gameplay/ping_pong/player_scores.cpp

```diff
@@ -11,7 +11,7 @@ using namespace ping_pong;
 int PlayerScoresSystem::s_FieldWidth = 20;
 int PlayerScoresSystem::s_FieldHeight = 10;
 Float32 PlayerScoresSystem::s_TileSize = 1.f;
-
+bool k = true;
```

Comment:
Please use more descriptive variable names. Names get stripped by the compiler anyway, and it helps everyone to know what `k` does.